src/Skoruba.IdentityServer4.Admin.Api/Program.cs

```diff
@@ -38,7 +38,9 @@ private static IConfiguration GetConfiguration(string[] args)
             var configurationBuilder = new ConfigurationBuilder()
                 .SetBasePath(Directory.GetCurrentDirectory())
                 .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
-                .AddJsonFile("serilog.json", optional: true, reloadOnChange: true);
+                .AddJsonFile($"appsettings.{environment}.json", optional: true, reloadOnChange: true)
```

Comment:
Do we really need this line ? I think apps already works with appsettings{environemnt}.json by default.

Reply:
Imho yes, this line is necessary as there is no "automagic" that is going on here, see https://github.com/skoruba/IdentityServer4.Admin/issues/549#issuecomment-603730495

Reply:
I don't think we need this line. If anyone wants a custom serilog configuration. The configuration should be mounted into the application. 

Define in your release pipeline to mount the serilog.json from the outside to the inside. The proposed line will not work with docker and kubernetes

Reply:
> I don't think we need this line. If anyone wants a custom serilog configuration. The configuration should be mounted into the application.
> 
> Define in your release pipeline to mount the serilog.json from the outside to the inside. The proposed line will not work with docker and kubernetes

Thanks for your feedback but i don't understand your instructions. What does "mounted into the application" actually mean and what "release pipeline"? Do you mean i should copy Serilog.Development.json to Serilog.json before releasing the application?

Reply:
if you look at the docker compose below
https://github.com/skoruba/IdentityServer4.Admin/blob/master/docker-compose.yml#L35

This give the flexibility for anyone to define serilog.json as they want and mount it from the host machine to the container

Reply:
In docker or kubernetes if you have multiple applications that required serilog you only want to have one config. In kubernetes you can use a config map or secret and mount those inside the pod. 

If it need to change you change the config map without having to redploy your application

Reply:
@ekjuanrejon So this PR is not necessary at all?

Reply:
To me it is not necessary but if you want I'd say if you still want to do it. do it in your fork

Reply:
So you want only "necessary" changes not improvements, that's sad as my approach also [allows config transformations](https://stackoverflow.com/questions/53208920/equivalent-to-app-config-transforms-for-net-core) via [SlowCheetah](https://marketplace.visualstudio.com/items?itemName=vscps.SlowCheetah-XMLTransforms), all effort in vain. Closing the PR.

Reply:
I am not the owner of the repo. I just gave a comment.

Reply:
The owner can very well decide to approve your pr

Reply:
@skoruba